tests/test_create_account.py

```diff
@@ -4,26 +4,29 @@
 import re
 
 import pytest
-from unittestzero import Assert
 
-from fxapom.fxapom import FxATestAccount
+from fxapom.fxapom import DEV_URL, FxATestAccount, PROD_URL
 
 
-@pytest.mark.nondestructive
 @pytest.mark.skip_selenium
 class TestCreateAccount(object):
 
-    def test_create_new_account_on_dev(self):
-        acct = FxATestAccount(base_url='https://www-dev.allizom.org').create_account()
-        Assert.true(acct.is_verified)
-        Assert.equal('https://stable.dev.lcip.org/auth/', acct.fxa_url)
+    def test_create_new_account_on_dev_by_default(self):
+        acct = FxATestAccount().create_account()
+        assert acct.is_verified
+        assert 'https://stable.dev.lcip.org/auth/' == acct.fxa_url
```

Comment:
Do we need to create an account here? It seems we're just testing the default value of the url argument. We could do this by just writing `assert DEV_URL == FxATestAccount().fxa_url`. This leads me to wonder why we can create an account object without creating an actual account - is there a use case for that?